README.md

```diff
@@ -26,7 +26,7 @@ Storage plugins allow Porter to store data, such as claims and credentials, in t
 The `azure.blob` plugin stores data in Azure Blob Storage. 
 
 1. Open, or create, `~/.porter/config.toml`.
-1. Add the following line to instruct Porter to use the azure blob storage plugin
+1. Add the following line activate the Azure blob storage plugin
```

Comment:
Oops, missing a `to` betwixt `line` and `activate`.

Reply:
100 internet points for using betwixt